pkg/runner/cluster_k8s.go

```diff
@@ -657,6 +656,24 @@ func (c *ClusterK8sRunner) ensureCollectOutputsPod(ctx context.Context, input *a
 	return nil
 }
 
+func (c *ClusterK8sRunner) getPodLogsStream(ctx context.Context, podName string) (io.ReadCloser, error) {
+	client := c.pool.Acquire()
+	defer c.pool.Release(client)
+
+	podLogOpts := v1.PodLogOptions{
+		LimitBytes: int64Ptr(10000000000), // 100mb
```

Comment:
Can we have more? 
Ideally having a const would be greatly appreciated

Reply:
we can have any value and yes we can have a `const`. But I'd suggest to make it configurable